test/UnitSPIRV.h

```diff
@@ -99,6 +99,7 @@ class WordVector {
 
 inline void PrintTo(const WordVector& words, ::std::ostream* os) {
   size_t count = 0;
+  const auto saved_flags = os->flags();
```

Comment:
FWIW, the style guide says to use "Use explicit formatting functions rather than stream manipulators or formatting flags to control formatting details such as number base, precision, or padding."
However, what are the formatting functions of which they speak, if not sprintf-like behaviour which is rather low level, or the output stream calls ??  Count me mystified.